lib/livebook/notebook/explore/intro_to_livebook.livemd

```diff
@@ -60,6 +60,32 @@ for _ <- 1..3 do
 end
 ```
 
+<!-- livebook:{"parent_index":2} -->
```

Comment:
In the loaded structure we point to parent using id, so it's safe to move/add sections. We don't persist ids, so here we store the corresponding index instead.

Reply:
What if we call it "parent_branch_index"? Just to make it clear this is branch related?

Reply:
"parent_branch" doesn't sound accurate though, could be more verbose with "parent_section_index", not sure if that helps though?

Reply:
Ah, I probably meant "branch_parent_index". Another option is to use nesting:

  "branch": {"parent_index": ...}

Thoughts?

Reply:
"branch_parent_index" sounds good, we could surely use an object, but I can't really think of any configuration we may want so it probably falls under YAGNI.

Reply:
As you prefer.